__tests__/apis.test.js

```diff
@@ -222,3 +222,80 @@ describe("POST a new comment to comments table", () => {
       });
   });
 });
+
+describe("8. PATCH /api/reviews/:review_id", () => {
+  test("status code 200, responds with an increased vote count", () => {
+    const newVote = { inc_votes: 10 };
+    return request(app)
+      .patch("/api/reviews/1")
+      .send(newVote)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.review).toMatchObject({
+          review_id: 1,
+          title: "Agricola",
+          category: "euro game",
+          designer: "Uwe Rosenberg",
+          owner: "mallionaire",
+          review_body: "Farmyard fun!",
+          review_img_url:
+            "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+          created_at: "2021-01-18T10:00:20.514Z",
+          votes: 11,
+        });
+      });
+  });
+
+  test("status code 200, responds with a decreased vote count", () => {
+    const newVote = { inc_votes: -99 };
+    return request(app)
+      .patch("/api/reviews/1")
+      .send(newVote)
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.review).toEqual({
+          review_id: 1,
+          title: "Agricola",
+          category: "euro game",
+          designer: "Uwe Rosenberg",
+          owner: "mallionaire",
+          review_body: "Farmyard fun!",
+          review_img_url:
+            "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+          created_at: "2021-01-18T10:00:20.514Z",
+          votes: -98,
+        });
+      });
+  });
+
+  test("400 - returns an error message if review_id is out of bounds", () => {
+    return request(app)
+      .patch("/api/reviews/999")
+      .send({ inc_votes: 15 })
+      .expect(404)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("review 999 does not exist");
+      });
+  });
+  
+  test("400 - returns an error message if review_id is not a number", () => {
+    return request(app)
+    .patch("/api/reviews/skfuhsuif")
+    .send({ inc_votes: 15 })
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("invalid id");
+      });
+  });
+
+  test("400 - returns an error message if  is not a number", () => {
+    return request(app)
+    .patch("/api/reviews/skfuhsuif")
+    .send({ inc_votes: 'votehehe' })
+      .expect(400)
+      .then(({ body: { msg } }) => {
+        expect(msg).toBe("votehehe is not a number");
+      });
+  });
+});
+
```

Comment:
This needs one more test to handle a missing request body. You have the error handling code for a missing inc_votes, but no corresponding test for it.